README.md

```diff
@@ -19,7 +19,7 @@ npm run generate -- --type=PRODUCT_SUBSCRIPTION
 ## With yarn
 yarn
 
-yarn generate --type=PRODUCT_SUBSCRIPTION
+yarn generate --type=PRODUCT_SUBSCRIPTION --template=js+react
```

Comment:
Can you add a section that lists out the available templates similar to the `Available extensions` section?